tree/20_cfe_basics/dry_run_lib.cf

```diff
@@ -56,77 +56,86 @@ bundle agent set_dry_run_mode(mode)
 #  - ... and all promise types that are not for the agent
 ###############################################################################
 
+body file control
+{
+   namespace => "bodydefault"; 
```

Comment:
This seems rather an odd name. Can't we use our standard conventions, and have something like "ncf_body_default"?